third_party/src/default/assets/js/main.js

```diff
@@ -431,21 +431,26 @@ function renderSimpleHTMLRecursive(obj, package, spacing) {
             if (window.location.href.indexOf('modules') == -1) {
                 href = 'modules/';
             }
-            if (window.location.href.indexOf('interfaces') > -1 ||
-                window.location.href.indexOf('assets') > -1 ||
-                window.location.href.indexOf('classes') > -1) {
+            if (window.location.href.indexOf('assets') > -1 ||
+                window.location.href.indexOf('classes') > -1 ||
+                window.location.href.indexOf('enums') > -1 ||
+                window.location.href.indexOf('interfaces') > -1
+            ) {
                 href = '../modules/';
             }
+            if(window.location.href.indexOf('modules') > -1){
+                href = "../modules/" + href;
```

Comment:
If this happens, what is the value of `href`? It seems to me like it would be undefined.

Reply:
`../modules/` and `/modules/` don't seem to be different, at least they can be accessed normally.

![2019-10-07-02](https://user-images.githubusercontent.com/24560368/71369449-c620d380-25e5-11ea-9413-23e1cf10eb0d.gif)